lambdas/build-user-identity/src/main/java/uk/gov/di/ipv/core/builduseridentity/BuildUserIdentityHandler.java

```diff
@@ -159,7 +167,12 @@ public APIGatewayProxyResponseEvent handleRequest(
 
             var contraIndicators = ciMitService.getContraIndicators(contraIndicatorsVc);
 
-            var vcs = verifiableCredentialService.getVcs(userId);
+            var vcs =
+                    configService.enabled(SESSION_CREDENTIALS_TABLE_READS)
+                            ? sessionCredentialsService.getLatestCredentialsByIssuer(
+                                    ipvSessionId, userId)
+                            : verifiableCredentialService.getVcs(userId);
```

Comment:
This makes me think - we should probably check with Orchestration and SPOT that sending two address VCs (in 6MFC) is desirable.

Otherwise we might need to drop the old address VC _somewhere_ in the journey...

Reply:
If we use this implementation as written here, we won't be sending two address VC?
Is 6MFC live yet?

Reply:
6MFC is planned to go live on Wednesday :)

Thinking about it after the weekend, I agree we shouldn't send a second address VC for now (as you've implemented). However it's worth noting, there have been definite rumblings about sending an 'address history' in some cases - which presumably _would_ include all address VCs - so we don't want to design ourselves into a corner.

As mentioned on the other PR, it feels like it'd better to just have the 'correct' set of VCs in the session store - if we no longer consider the old address as part of the user's identity.